lib/solidus_affirm/engine.rb

```diff
@@ -8,5 +8,11 @@ class Engine < Rails::Engine
     config.generators do |g|
       g.test_framework :rspec
     end
+
+    config.autoload_paths += %W(#{config.root}/lib)
+
+    initializer "spree.gateway.payment_methods", after: "spree.register.payment_methods" do |app|
```

Comment:
Prefer telling users how to do this for their application as part of the engines install instructures. Or the install task can add an initializer. Users have no way to remove the payment method otherwise.